packages/fixie/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './client.js';
+export * from './fixie-embed.js';
```

Comment:
I'm not sure about combining `fixie` and `fixie/web` because it will require React everywhere